action.yml

```
@@ -17,6 +17,9 @@ inputs:
   requireScope:
     description: "Configure that a scope must always be provided."
     required: false
+  disallowScopes:
+    description: 'Configure a list of disallowed scopes to be used on a PR.'
```

Synced over the description from the README:

```suggestion
    description: 'Configure which scopes are disallowed in PR titles.'
```